src/App.jsx

```diff
@@ -6,14 +6,18 @@ import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
 
 function App() {
-  const[newTodo, setNewTodo] = useState("");
+  const[todoList, setTodoList] = useState([]);
+  function addTodo(newTodo){
+    setTodoList((prevTodoList) => [...prevTodoList, newTodo]);
+  }
+  
    return (
   <>
     <div>
       <h1>Todo List</h1>
-        <TodoList />
-        <AddTodoForm onAddTodo={setNewTodo}/>
-        <p>{newTodo}</p>
+        <TodoList todoList={todoList}/>
+        <AddTodoForm onAddTodo={addTodo}/>
+
```

Comment:
@A-Kaliexe Switch places between the `TodoList` and `AddTodoForm`. The Form should come first so that it's always at the top, instead of getting pushed towards the bottom with the addition of every new todo item